src/components/navigation/navigationHeader.svelte

```diff
@@ -0,0 +1,14 @@
+<header class="leo-navigation-header">
```

Comment:
I don't know if this should use the `header` element - it isn't necessarily going to contain the header for, say an article, but rather for the site (with the nav links).

Don't know well enough to say either way :laughing:

Reply:
From seeing it's use, I believe the intent is that this would be the `header` for the page.